libs/models/src/lib/file.ts

```diff
@@ -0,0 +1,44 @@
+import mongoose from 'mongoose';
+import { IFile } from '@kol-amelamdim/types';
+
+const FilesSchema = new mongoose.Schema(
+  {
+    key: {
+      type: String,
+      required: true,
+      unique: true,
+    },
+    category: {
+      type: String,
+      required: true,
+    },
+    name: {
+      type: String,
+      required: true,
+    },
+    size: {
+      type: String,
+      required: true,
+    },
+    author: {
+      type: String,
+      required: true,
```

Comment:
Let's add more validation for author like 'minlength, maxlength, trim:true' etc.

Reply:
for now im just passing hard coded string 'the author' lets decide together how to set the author and then all of the requirement..